integration_test/install_test.go

```diff
@@ -25,3 +34,42 @@ func TestKrewInstall(t *testing.T) {
 	test.WithIndex().Krew("install", validPlugin).RunOrFailOutput()
 	test.AssertExecutableInPATH("kubectl-" + validPlugin)
 }
+
+func TestKrewInstall_Manifest(t *testing.T) {
+	skipShort(t)
+
+	test, cleanup := NewTest(t)
+	defer cleanup()
+
+	test.Krew("install",
+		"--manifest", filepath.Join("testdata", validPlugin+constants.ManifestExtension)).
+		RunOrFail()
+	test.AssertExecutableInPATH("kubectl-" + validPlugin)
+}
+
+func TestKrewInstall_ManifestAndArchive(t *testing.T) {
+	skipShort(t)
+
+	test, cleanup := NewTest(t)
+	defer cleanup()
+
+	test.Krew("install",
+		"--manifest", filepath.Join("testdata", fooPlugin+constants.ManifestExtension),
```

Comment:
You don't need to address this but: I feel like these small constants are hurting the readability quite a bit, unless needed, we can probably avoid in the future.